howto/configure-mtls/README.md

```diff
@@ -0,0 +1,198 @@
+# Setup and Configure Mututal TLS
+
+Dapr supports in-transit encryption of communication between Dapr instances using Sentry, a central Certificate Authority.
+
+Dapr allows operators and developers to bring in their own certificates, or let Dapr automatically create and persist self signed root and issuer certificates.
+
+For detailed information on mTLS, visit the concepts section [here](../../concepts/security/security.md).
+
+If custom certificates have not been provided, Dapr will automaically create and persist self signed certs valid for one year.
+In Kubernetes, the certs will be persisted to a secret that resides in the namespace of the Dapr system pods, accessible only to them.
+In Self Hosted mode, the certs will be persisted to disk. More information on that is shown below.
+
+## Sentry Configuration
+
+mTLS settings reside in a Dapr configuration file.
+The following file shows all the available settings for mTLS in a configuration resource:
+
+```
+apiVersion: dapr.io/v1alpha1
+kind: Configuration
+metadata:
+  name: default
+spec:
+  mtls:
+    enabled: true
+    workloadCertTTL: "24h"
+    allowedClockSkew: "15m"
+```
+
+The file here shows the default configuration settings. The examples below will show you how to change and apply this configuration to Sentry in Kubernetes and Self hosted modes.
```

Comment:
"below show you how" - I should refrain from using the word "will" and use the present tense, since it works now not in the future

Reply:
Changed